### ISSUE TITLE
[BUG] Unable to install slack app when scopes are exceeding 255 characters

### ISSUE THREAD

**Issue by stephanebenayoun** [seratch/slack-bolt-extensions#10]
The fields `botScopes` and `userScopes` are currently defined as DataTypes.STRING (varchar) [here](https://github.com/seratch/slack-bolt-extensions/blob/6dd7a29e305510038054ec9e55c60bded575c6fe/packages/bolt-sequelize/src/SlackAppInstallation.ts#L76C7-L76C16), they should be defined with another type (probably TEXT) as they can potentially exceed 255 characters.

When scopes are larger than 255 characters, the slack app installation failed with the following error:
```
/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:514:28)
      at addChunk (node:internal/streams/readable:343:12)
      at readableAddChunk (node:internal/streams/readable:316:9)
      at Readable.push (node:internal/streams/readable:253:10)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    length: 99,
    severity: 'ERROR',
    code: '22001',
    detail: undefined,
    hint: undefined,
    position: undefined,
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'varchar.c',
    line: '632',
    routine: 'varchar',
    sql: 'INSERT INTO "slack_app_installations" ("id","clientId","appId","teamId","teamName","botToken","botId","botUserId","botScopes","userId","isEnterpriseInstall","tokenType","installedAt","createdAt","updatedAt") VALUES (DEFAULT,$1,$2,$3,$4,$5,$6,$7,$8,$9,$10,$11,$12,$13,$14) RETURNING "id","clientId","appId","enterpriseId","enterpriseName","enterpriseUrl","teamId","teamName","botToken","botId","botUserId","botScopes","botRefreshToken","botTokenExpiresAt","userId","userToken","userScopes","userRefreshToken","userTokenExpiresAt","incomingWebhookUrl","incomingWebhookChannel","incomingWebhookChannelId","incomingWebhookConfigurationUrl","isEnterpriseInstall","tokenType","installedAt","createdAt","updatedAt";',
    parameters: [
      '***',
      ''***',
      ''***',
      ''***',
      ''***,
      ''***',
      ''***',
    'channels:history,channels:read,chat:write,chat:write.customize,chat:write.public,commands,groups:history,im:history,im:write,mpim:history,users:read,users:read.email,reactions:write,links:read,links:write,remote_files:write,remote_files:read,reactions:read',
      ''***',
      false,
      'bot',
      '2024-01-17 08:40:56.366 +00:00',
      '2024-01-17 08:40:56.366 +00:00',
      '2024-01-17 08:40:56.366 +00:00'
    ]
  },
  original: error: value too long for type character varying(255)
```

**Comment by seratch** [comment]
Hi, thanks for taking the time to report the issue. Could you send a PR to resolve it if you have time for it?

**Comment by seratch** [comment]
Fixed by #11

**Comment by seratch** [comment]
Thank you for fixing the issue! A new patch version (1.0.5) including the fix is now available.
